UIKitTests/UIView/UIViewTests+convert.swift

```diff
@@ -243,6 +243,36 @@ class UIViewPointConversionTests: XCTestCase {
         XCTAssertEqual(convertedPoint1, convertedPoint2)
         XCTAssertEqual(convertedPoint1, CGPoint(x: 50, y: 50))
     }
+
+    func testConversionWithMultipleSubviewsBounds() {
```

Comment:
It would be great if you could add the `transform`s back in here but commented out (we could even keep of them for now I think). It would be really nice to have a clear place to work on when we come back to this someday